### PR TITLE
Implement register

### PR DESCRIPTION
**Pull request by MelvinPeepers**
done with the barebones for signup page, should be working

### CODE REVIEW

File: app-sleeptracker/src/App.js [hunk 1]
```diff
@@ -3,13 +3,15 @@ import { Route } from "react-router-dom";
 import PrivateRoute from "./components/PrivateRoute";
 import Dashboard from "./components/Dashboard";
 import Login from "./components/Login/Login";
+import Reg from "./components/registration/Reg";
```
Comment (depadiernos) [inline app-sleeptracker/src/App.js]: Not wrong, but I recommend just using 'Register' or 'SignUp' to make the component more explicit at what it is. No need to change. Just my 2 cents.

File: app-sleeptracker/src/components/registration/Reg.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      })
+      .catch(error => {
+        console.log(error);
+      });
+  };
+
+  render() {
+    const { username, password, birthdate } = this.state;
+    const { signingUp } = this.props;
+    return (
```
Comment (depadiernos) [inline app-sleeptracker/src/components/registration/Reg.js]: Are you exposing errors? Like "account already exists"? Similar to 'Invalid Credentials" in the Login component.

**Comment by depadiernos** [comment]
Soon as we get this merged, we need to add a _redirects file in your build folder with the following redirect rule:

`/* /index.html 200`

File: app-sleeptracker/src/App.js (hunk 1 above)
Comment (MelvinPeepers) [reply app-sleeptracker/src/App.js]: Thanks, I'll do that.

File: app-sleeptracker/src/components/registration/Reg.js (hunk 2 above)
Comment (MelvinPeepers) [reply app-sleeptracker/src/components/registration/Reg.js]: hmm not sure, we'll need to discuss this.

**Pull request merged**